src/cli/commands/test/cache.rs

```diff
@@ -1,17 +1,27 @@
 pub mod cache {
```

Comment:
I think you can get rid of pub mod cache here, so that you won't have `use crate::cli::commands::test::cache::cache::` later on

Reply:
Ah nice! This is in the next PR